### PR TITLE
add option for binding exporter to ip

### PR DESCRIPTION
**Pull request by and0x000**
binds to localhost only by default, preventing unintended exposure of this service

closes https://github.com/discourse/prometheus_exporter/issues/101

### CODE REVIEW

**Comment by and0x000** [comment]
Feel free to edit the PR. Setting `PrometheusExporter::DEFAULT_BIND_ADDRESS` to '*' instead of 'localhost' should keep the current behavior, where it binds to all addresses. However, I honestly believe, defaulting to 'localhost' would be best practice.

**Comment by SamSaffron** [comment]
OK I can accept the behavior change, it is breaking so we will need to be careful here.

Can you amend the README and CHANGELOG as well in light of this?

File: README.md [hunk 1]
```diff
@@ -62,8 +62,9 @@ require 'prometheus_exporter/server'
 require 'prometheus_exporter/client'
 require 'prometheus_exporter/instrumentation'
 
+# bind is the address, on which the webserver will listen
 # port is the port that will provide the /metrics route
-server = PrometheusExporter::Server::WebServer.new port: 12345
+server = PrometheusExporter::Server::WebServer.new bind: localhost, port: 12345
```
Comment (SamSaffron) [inline README.md]:
```suggestion
server = PrometheusExporter::Server::WebServer.new bind: 'localhost', port: 12345
```

**Comment by SamSaffron** [comment]
OK lets give this a shot! Thanks!

**Pull request merged**